src/nn/graph_builder.ts

```diff
@@ -75,16 +76,40 @@ export enum MLAutoPad {
  * [spec](https://webmachinelearning.github.io/webnn/#dictdef-mlconv2doptions)
  */
 export interface MLConv2dOptions {
+  padding?: [number, number, number, number];
+  strides?: [number, number];
+  dilations?: [number, number];
+  outputPadding?: [number, number];
```

Comment:
There isn't `outputPadding` option for [`MLConv2dOptions`](https://webmachinelearning.github.io/webnn/#dictdef-mlconv2doptions), please remove it.